### DISCUSSION TITLE
[ 功能建议 ]建议优化ui界面、添加异步导出功能、实现ddl和dml分离

### DISCUSSION THREAD

**Discussion by czxin788** [hhyo/Archery#1608] [Ideas]
### 目前遇到的问题/使用障碍
发现一款审核平台，感觉ui做的好看，和navicat界面 似的，而且把dml和ddl分开审批，我觉得这样符合我们现在需求。DBA只审核DDL语句；DML语句只需要RD组长审批即可
这款产品叫sqldev，
https://github.com/shuaninfo/sqldev
### 希望如何解决/实现它
1、个人觉得archery的界面不美观；
2、ddl和dml分离审批；
3、建议添加异步导出功能。当数据量超过5000条时，提交工单审批，并转成离线下载excel的方式。
参见 http://sqldev.info/help-document/%E5%B7%A5%E4%BD%9C%E5%8C%BA/%E5%BC%82%E6%AD%A5%E5%AF%BC%E5%87%BA.html

### 其他信息
![image](https://user-images.githubusercontent.com/13861904/149648764-60253957-70ec-4a5e-b3db-b77ebd0bcbea.png)

**Comment by hhyo** [comment]
感谢关注和建议，如果确实无法适配您的需求，可以参与二次开发，或者看是否有其他社区朋友愿意贡献相关改进，项目无专职的维护人员和开发团队，短期内除bug修复外无更多的功能迭代计划

**Comment by SteveYangyang** [comment]
> 
这个导出功能在我的二开里面实现了！邮箱发送下载连接。然后点击下载即可。
![image](https://user-images.githubusercontent.com/26180009/150267588-5bf1c002-9494-4e77-9a11-0006a94cc8b5.png)

**Comment by czxin788** [comment]
太棒了。
这个能走工单审批吗。
超过5000条的数据，必须工单审批通过后，才能离线下载。
这样也能保护数据安全。
对于离线下载的最好有个审计记录，将来出现数据安全问题，也方便排查问题。看谁导出了大量数据出去，方便追责

**Comment by LeoQuote** [comment]
> > 
> 
> 这个导出功能在我的二开里面实现了！邮箱发送下载连接。然后点击下载即可。 ![image](https://user-images.githubusercontent.com/26180009/150267588-5bf1c002-9494-4e77-9a11-0006a94cc8b5.png)

@SteveYangyang 
欢迎pr
